website/database.py

```diff
@@ -730,3 +732,17 @@ def to_year_week(self, d):
     def get_username_role(self, username):
         role = "teacher" if USERS.get({"username": username}).get("teacher_request") is True else "student"
         return role
+
+    def test_group_by_username(self, username):
+        test_group = TEST_GROUP.get({"username": username})
+        if test_group:
+            return test_group["test_group"]
+        return None
```

Comment:
This can be simplified with `return test_group.get('test_group')`.

Reply:
done.